CONTRIBUTING.md

```diff
@@ -33,4 +33,26 @@ In order to develop it locally we suggest to use [polyserve](https://npmjs.com/p
 3. Commit your changes: `git commit -m 'Add some feature'`
 4. Push to the branch: `git push origin my-new-feature`
 5. Open corresponding issue if needed
-6. Submit a pull request :D
\ No newline at end of file
+6. Submit a pull request :D
+
+## Unit tests
+
+All Juicy custom elements projects use [`web-component-tester`](https://github.com/Polymer/web-component-tester) for unit tests.
+The [`polyserve`](https://github.com/PolymerLabs/polyserve) utility is helpful for [running tests in the browser](#developing-the-element).
+
+### Running element unit tests from CLI
+
+To run the element unit tests from CLI, you need to:
+
+0.  Install `web-component-tester` globally: `npm install -g web-component-tester`
+1.  Clone the element repo.
+2.  Install the dependencies. `bower install`
+3.  Run the tests: `wct`
```

Comment:
Can you pls add a note:

(be patient - the first run might take 5 minutes to download the Selenium framework)

Reply:
Done